apps/sms/views/shared/test/unit/contacts_test.js

```diff
@@ -517,16 +317,27 @@ suite('Contacts', function(done) {
   suite('Contacts.findByPhoneNumber', function() {
 
     test('removes spaces', function(done) {
-      var mozContacts = navigator.mozContacts;
+      // Unwraps the sinon.stub
+      navigator.mozContacts.find.restore();
+      // Spy for navigator.mozcontacts
+      var spy = sinon.spy(navigator.mozContacts, 'find');
 
       Contacts.findByPhoneNumber('+33 1 23 45 67 89').then(function(contacts){
-        assert.equal(
-          mozContacts.mHistory[0].filter.filterValue, '+33123456789'
+        sinon.assert.calledWith(
+          spy, sinon.match({ filterValue: '+33123456789' })
```

Comment:
I think that if you use `withArgs` properly when defining the stub, you won't need to check the spy afterwards; but we can check if the returned contacts are correct :)